daft/daft.pyi

```diff
@@ -934,7 +934,7 @@ class PhysicalPlanScheduler:
     def num_partitions(self) -> int: ...
     def to_partition_tasks(
         self, psets: dict[str, list[PartitionT]], is_ray_runner: bool
-    ) -> physical_plan.MaterializedPhysicalPlan: ...
+    ) -> physical_plan.InProgressPhysicalPlan: ...
```

Comment:
I think this was just typed wrongly from the beginning, not sure why our typechecks didn't catch it.